packages/langchain/src/orchestration/util.ts

```diff
@@ -320,18 +317,12 @@ export function mapOutputToChatResult(
   };
 }
 
-type ToolDefinitionLike = Pick<ToolDefinition, 'type'> & {
-  function: Omit<FunctionDefinition, 'parameters'> & {
-    parameters?: FunctionParameters;
-  };
-};
-
 /**
  * @internal
  */
 export function isToolDefinitionLike(
```

Comment:
[nitpick] I would remove `Like` here since the type no longer exists and the name feels confusing otherwise.